frontend/src/component/feature/FeatureStrategy/FeatureStrategyMenu/FeatureStrategyMenuCards/FeatureStrategyMenuCards.tsx

```diff
@@ -10,6 +10,7 @@ interface IFeatureStrategyMenuCardsProps {
     projectId: string;
     featureId: string;
     environmentId: string;
+    onlyReleasePlans: boolean;
```

Comment:
This component is used in literally one place so rather than duplicate it, I've just given it the option to hide the other strategy stuff